cot/src/handler.rs

```diff
@@ -116,79 +120,104 @@ macro_rules! impl_request_handler_from_request {
-impl_request_handler_from_request!((P1), P2, P3, P4, P5, P6);
-impl_request_handler_from_request!(P1, (P2), P3, P4, P5, P6);
-impl_request_handler_from_request!(P1, P2, (P3), P4, P5, P6);
-impl_request_handler_from_request!(P1, P2, P3, (P4), P5, P6);
-impl_request_handler_from_request!(P1, P2, P3, P4, (P5), P6);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, (P6));
-impl_request_handler_from_request!((P1), P2, P3, P4, P5, P6, P7);
-impl_request_handler_from_request!(P1, (P2), P3, P4, P5, P6, P7);
-impl_request_handler_from_request!(P1, P2, (P3), P4, P5, P6, P7);
-impl_request_handler_from_request!(P1, P2, P3, (P4), P5, P6, P7);
-impl_request_handler_from_request!(P1, P2, P3, P4, (P5), P6, P7);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, (P6), P7);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, (P7));
-impl_request_handler_from_request!((P1), P2, P3, P4, P5, P6, P7, P8);
-impl_request_handler_from_request!(P1, (P2), P3, P4, P5, P6, P7, P8);
-impl_request_handler_from_request!(P1, P2, (P3), P4, P5, P6, P7, P8);
-impl_request_handler_from_request!(P1, P2, P3, (P4), P5, P6, P7, P8);
-impl_request_handler_from_request!(P1, P2, P3, P4, (P5), P6, P7, P8);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, (P6), P7, P8);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, (P7), P8);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, P7, (P8));
-impl_request_handler_from_request!((P1), P2, P3, P4, P5, P6, P7, P8, P9);
-impl_request_handler_from_request!(P1, (P2), P3, P4, P5, P6, P7, P8, P9);
-impl_request_handler_from_request!(P1, P2, (P3), P4, P5, P6, P7, P8, P9);
-impl_request_handler_from_request!(P1, P2, P3, (P4), P5, P6, P7, P8, P9);
-impl_request_handler_from_request!(P1, P2, P3, P4, (P5), P6, P7, P8, P9);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, (P6), P7, P8, P9);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, (P7), P8, P9);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, P7, (P8), P9);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, P7, P8, (P9));
-impl_request_handler_from_request!((P1), P2, P3, P4, P5, P6, P7, P8, P9, P10);
-impl_request_handler_from_request!(P1, (P2), P3, P4, P5, P6, P7, P8, P9, P10);
-impl_request_handler_from_request!(P1, P2, (P3), P4, P5, P6, P7, P8, P9, P10);
-impl_request_handler_from_request!(P1, P2, P3, (P4), P5, P6, P7, P8, P9, P10);
-impl_request_handler_from_request!(P1, P2, P3, P4, (P5), P6, P7, P8, P9, P10);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, (P6), P7, P8, P9, P10);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, (P7), P8, P9, P10);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, P7, (P8), P9, P10);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, P7, P8, (P9), P10);
-impl_request_handler_from_request!(P1, P2, P3, P4, P5, P6, P7, P8, P9, (P10));
+#[rustfmt::skip]
+macro_rules! handle_all_parameters {
```

Comment:
Looks like we could simplify `crate::openapi::impl_as_openapi_operation` invocations with this!